### DISCUSSION TITLE
CLI flags or options to add eslint + prettier to `npx nuxi init`

### DISCUSSION THREAD

**Discussion by oosame** [nuxt/nuxt#19211] [Questions]
can we have these, please? I tried doing it myself but I'm often lost when setting everything up with other tools I use and VSCode/nvim and I would like to make use of nuxt 3

**Comment by ndragun92** [comment]
I know it might be outside of ur question and suggestion but I have made for myself own template repo that I use for all new projects

https://github.com/ndragun92/template-nuxt3
